oeps/oep-0001.rst

```diff
@@ -45,163 +61,211 @@ OEP Types
+will never be selected as the Arbiter of that OEP.
 
 The Arbiter will be the person making the final decision on whether the OEP 
 should be Accepted, and as such, the Arbiter should be knowledgeable about 
 the contents of the proposal, and willing to listen to arguments both for 
 and against it by the rest of the community.
 
-The Arbiter is also responsible for helping the Author move the proposal
+The Arbiter is also responsible for helping the Authors move the proposal
 through the OEP process, providing technical and process expertise as needed.
-The Arbiter also assists the Author to solicit feedback from the community on
+The Arbiter also assists the Authors to solicit feedback from the community on
 the OEP, and for helping to move the OEP towards a final decision (whether that
 decision is Accepted, Rejected, or Deferred). The Arbiter (in discussion with 
-the Author) can merge an in-progress OEP (if it has reached a stage of relative
+the Authors) can merge an in-progress OEP (if it has reached a stage of relative
 stability) to allow for additional incremental updates.
 
 Finally, the Arbiter is responsible for the decision to transfer an OEP if the
-original Author has become unresponsive (as described in `Transferring OEP
+original Authors have become unresponsive (as described in `Transferring OEP
 Ownership`_).
 
-Before Submitting an OEP
-------------------------
+Architecture Team
+-----------------
 
-The OEP process begins with a new idea for Open edX. It is highly recommended
-that a single OEP contain a single key proposal or new idea. Small enhancements
-or patches often don't need an OEP and can be injected into the Open edX
-development workflow with a patch submission.
+The `edX Architecture Team`_ is accountable for assigning Arbiters to incoming
+Draft OEPs and revived OEPs that need a new Arbiter (if the original Arbiter is no
+longer available). The team can also be a resource to help/advise the Arbiter with
+the OEP process.
 
-The OEP Author should first attempt to ascertain whether the idea is appropriate 
-for an OEP.  Posting to the `edx-code`_ mailing list is the best way to go about 
-this.
+.. _edX Architecture Team: https://openedx.atlassian.net/wiki/spaces/AC/pages/439353453/Architecture+Team
+
+OEP Workflow
+============
+
+.. contents::
+  :local:
+  :depth: 2
+
+Submitting an OEP
+-----------------
+
+Step 1. Scope your idea
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The OEP process begins with a new idea for Open edX. Unless it is a Best
+Practice, a single OEP should contain a single key proposal or new idea. Small
+enhancements or patches often don't need an OEP and can be injected into the Open
+edX development workflow with a patch submission.
+
+Step 2. Vet your idea
+~~~~~~~~~~~~~~~~~~~~~
+
+OEP Authors should first attempt to ascertain whether the idea is appropriate 
+for an OEP. Posting to the `open-edx-proposals Slack channel`_ is the best way
```

Comment:
Are there any persons or a group we should DM in the #open-edx-proposals slack channel when we post an idea?

Reply:
Good idea.  We created an @oep-team slack group for this - a group who can help with vetting an idea.  I'll update the doc.